examples/nlp/dialogue/dialogue.py

```diff
@@ -121,8 +124,9 @@ def main(cfg: DictConfig) -> None:
 
     if cfg.do_training:
         trainer.fit(model)
-        if cfg.model.nemo_path:
+        if cfg.model.nemo_path and not os.path.exists(cfg.model.nemo_path):
```

Comment:
If a new model just finished training, and has the same name as an existing model, instead of just not saving the new model, shouldn't a warning be raised and the new model saved with a slightly different name? e.g. with (1) appended to the end or something?

Reply:
Sure makes sense!